README.md

```diff
@@ -14,3 +14,8 @@ password:example_pass
 ```
 - Fetch the latest code with `git pull`
 - Update PyPI with `python setup.py sdist upload`
+
+## Running the tests
+
+- python setup.py install
+- python setup.py test
```

Comment:
It's preferable to invoke nose based tests with `nosetests` instead of with setuptools.